pkgs/applications/video/mpv/default.nix

```
@@ -82,7 +82,20 @@ let
   inherit (darwin.apple_sdk_11_0.frameworks)
     AVFoundation CoreFoundation CoreMedia Cocoa CoreAudio MediaPlayer Accelerate;
   luaEnv = lua.withPackages (ps: with ps; [ luasocket ]);
-in stdenv.mkDerivation (finalAttrs: {
+
+  overrideSDK = platform: version:
+    platform // lib.optionalAttrs (platform ? darwinMinVersion) {
+      darwinMinVersion = version;
+    };
+
+  stdenv' = if swiftSupport && stdenv.isDarwin && stdenv.isx86_64
+    then stdenv.override (old: {
+      buildPlatform =  overrideSDK old.buildPlatform  "10.15";
+      targetPlatform = overrideSDK old.targetPlatform "10.15";
+      hostPlatform =   overrideSDK old.hostPlatform   "10.15";
```

Too much whitespace (editorconfig should catch these):

```suggestion
      buildPlatform = overrideSDK old.buildPlatform  "10.15";
      targetPlatform = overrideSDK old.targetPlatform "10.15";
      hostPlatform = overrideSDK old.hostPlatform   "10.15";
```

---

I fixed the whitespace and the ordering. I also removed extra the whitespace before the version numbers.

> editorconfig should catch these

It apparently passed the editorconfig check. I don’t see anything in it that would enforce trimming whitespace padding (just trimming it from the end of the line).